### DISCUSSION TITLE
Steam Input vs AntiMicroX

### DISCUSSION THREAD

**Discussion by jtkellys** [AntiMicroX/antimicrox#915] [General]
Just a couple of questions:

1) Do the paddles show up on AntiMicroX? If so, is there anything special that needs to be done first? 

2) What benefits does AntiMicroX have over Steam Input?

**Comment by pktiuk** [comment]
Ad1. They should show up

Ad. 2 TBH I don't know. Since I bought steam deck I don't find AntiMicroX as useful as earlier. For now AntiMicro has a clear advantage in advanced mappings, when you can map button to selected combination of actions like pressing something, waiting, running a command etc.
